File: src/app/view/FunctionsController.java
package app.view;

import javafx.fxml.FXML;

public class FunctionsController {
    @FXML
    private void clickTextBtn() {
        MainController.changeToTextPane();
    }

    @FXML
    private void clickIdentityBtn() {
        MainController.changeToIdentityPane();
    }

    @FXML
    private void clickFileBtn() {
        MainController.changeToFilePane();
    }
}

File: src/app/view/FunctionsController.kt
package app.view

import javafx.fxml.FXML

class FunctionsController {
    @FXML
    private fun clickTextBtn() {
        MainController.changeToTextPane()
    }

    @FXML
    private fun clickIdentityBtn() {
        MainController.changeToIdentityPane()
    }

    @FXML
    private fun clickFileBtn() {
        MainController.changeToFilePane()
    }
}
